cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/template/etl/batch/BatchETLDBAdapterTest.java

```diff
@@ -270,6 +269,73 @@ public void testDBSource() throws Exception {
     Assert.assertEquals(clobData, Bytes.toString(row2.get("CLOB_COL"), 0, clobData.length()));
   }
 
+  @Test(expected = Exception.class)
+  @Category(SlowTests.class)
+  public void testBadConnSource() throws Exception {
```

Comment:
This test is missing some verification. I think just expecting ``Exception`` to be thrown may not be sufficient here, because, any sublass of ``Exception`` could be thrown at any LOC in this test?

Can we instead perhaps make sure that the job did not start? Or that there we sent some data from the source but there was no data in the ``Table`` after the test?